packages/cdktf-cli/bin/cmds/helper/init.ts

```diff
@@ -15,6 +15,19 @@ import { Errors } from "../../../lib/errors";
 import { convertProject, getTerraformConfigFromDir } from "@cdktf/hcl2cdk";
 import { execSync } from "child_process";
 import { sendTelemetry } from "../../../lib/checkpoint";
+import * as crypto from "crypto";
+
+function getProjectId(
+  projectName: string,
+  projectDescription: string,
+  projectPath = process.cwd()
+) {
+  const hash = crypto.createHash("md5");
+  hash.update(projectName);
+  hash.update(projectDescription);
+  hash.update(projectPath);
```

Comment:
What's the reason for hashing these values, rather than generating a uuid?

Reply:
There is none, could do either way

Reply:
I think a uuid would make more sense and is easier in this case